### ISSUE TITLE
ValidationError: instance.address is not exactly one from <xAddress>,<classicAddress>

### ISSUE THREAD

**Issue by mklem92** [ripple/ripple-lib#1082]
I am having issues when I am deploying my Angular 8 app to firebase. The xAddress or classicAddress seem to fail json schema validation but only once my app is deployed the firebase. It is working as expected with localhost though.

![validation_error](https://user-images.githubusercontent.com/26738347/68444494-b8281680-0222-11ea-8e1c-03c74efaf5dd.PNG)

I have tried a very simple example in my app to try and console log the balance of an XRP address

![example_1](https://user-images.githubusercontent.com/26738347/68444516-cd04aa00-0222-11ea-85e3-0ea368b4f8b8.PNG)

![example_2](https://user-images.githubusercontent.com/26738347/68444526-d3932180-0222-11ea-9422-c00a337b9c26.PNG)

I am certain this code did work a few weeks ago so I tried downgrading to 1.3.4 but received a similar validation error that address is not of 'address' format.

I ran the address through the same regex pattern that is occurring in the schema validation and it is definitely valid.  Any thoughts on what the issue may be?

**Comment by FKSRipple** [comment]
Can you share an address that isn't working in your hosted app? Also, is this running server side in Node.js, or client side in the browser?

This sounds like it may be specific to your application/hosting env, which will make it hard for us to debug remotely. Your best bet may be to add some `console.log()` error logging into the installed package code to see what is going wrong, or at least find a clue that could help us debug.

/cc @intelliot who may know more

**Comment by mklem3** [comment]
It appears that all address are failing for some reason. It's running from the client side, just called directly inside an Angular service. 

I might try comparing configs between my new project and the old. I have definitely had it working in the past but decided to recently start fresh.

I'll try and do the comparison today and I will let you know my findings.

**Comment by FKSRipple** [comment]
Great, let me know what you find. If you can get a consistent reproduction, please share and I can help take a look as well.

**Comment by mklem92** [comment]
Okay now that I have gone through it again I have started to remember some of the issues I ran into the first time I created my project. For some reason there are some issues with @angular/cli and Angular6+ versions seem to have some issues with stream and crypto modules.

I had to change the angular builder to use @angular-builders/custom-webpack:browser. This allows you to add some extra overrides to the webpack build process. I then added the extra-webpack.config file to get stream module working correctly. my extra-webpack.config looks like this :

`module.exports = {
  node: {
    crypto: true,
    http: true,
    https: true,
    os: true,
    vm: true,
    stream: true
  }
};
`
